app/assets/javascripts/components/reward_page_components/StudentFinalRewardPage.jsx

```diff
@@ -3,17 +3,17 @@ class StudentFinalRewardPage extends React.Component {
     return (
       <div className='section'>
         <p className='flow-text'>
-          Heel erg bedankt dat je meedeed aan ons onderzoek! Door jouw deelname kunnen wij onze webapp zo verbeteren
-          dat deze veel beter zal aansluiten aan de wensen van toekomstige deelnemers. Zodra de gegevens bij ons
-          binnen zijn ontvangt jouw S-team begeleider jouw beloning en kan jij je beloning daar dus ophalen.
+          Heel erg bedankt voor je inzet voor dit onderzoek!
         </p>
         <p className='flow-text'>
-           In totaal heb je €{this.props.earnedEuros} verdiend.
+          In totaal heb je €{this.props.earnedEuros} verdiend.
+          We zullen dit bedrag overmaken op IBAN nummer: {this.props.iban}.
```

Comment:
ergens is iban nummer trouwens dubbel op:

Internationaal Bank Account Nummer

Reply:
Fixed.